modules/core/src/main/scala/lucuma/odb/api/schema/TargetMutation.scala

```diff
@@ -149,22 +149,26 @@ trait TargetMutation extends TargetScalars {
 
   implicit val InputObjectEditTargetInput: InputObjectType[TargetModel.Edit] = {
 
-      // Not able to derive this for some reason, TBD.
+    // Not able to derive this for some reason, TBD.
 //    deriveInputObjectType[TargetModel.Edit](
 //      InputObjectTypeName("EditTargetInput"),
```

Comment:
I haven't figured this out yet.  The error message is `not found: value Edit`, though the same construct works everywhere else and I can't spot any difference.  I assume it has something to do with macros because `TargetModel.Edit` is visible just below in the non-macro version.